eoscostarica/lifebank
# feat(twitter): automatic tweets

Resolves #661 

### What does this PR do?
Automatic Tweets when adding new lifebanks or sponsors (accept contracts)
in the .env.example I added new variables for the twitter bot you can use my credentials for the time being
https://twitter.com/edgarparra2

### How should this be tested?
- Make run
- add new sponsor/lifebank
- Check if my profile posted a twit
